src/Tests/AuthoringWinUITest/AuthoringWinUITest/App.xaml.cpp

```diff
@@ -21,7 +26,7 @@ using namespace AuthoringWinUITest::implementation;
 App::App()
 {
     InitializeComponent();
-    Suspending({ this, &App::OnSuspending });
+    // Suspending({ this, &App::OnSuspending });

```

Comment:
intentional?

Reply:
I think this API changed in the winui changed, and I didn't find the alternative. Will find a better workaround